qa/rpc-tests/sc_big_block.py

```diff
@@ -140,8 +141,8 @@ def advance_sidechains_epoch(num_of_scs):
         certMcTest = CertTestUtils(self.options.tmpdir, self.options.srcdir, CERT_PROVING_SYSTEM)
         cswMcTest = CSWTestUtils(self.options.tmpdir, self.options.srcdir, CSW_PROVING_SYSTEM)
 
-        certVk = certMcTest.generate_params('scs', 'cert', CERT_NUM_CONSTRAINTS, SEGMENT_SIZE)
-        cswVk  = cswMcTest.generate_params('scs', 'csw', CSW_NUM_CONSTRAINTS, SEGMENT_SIZE)
+        certVk = certMcTest.generate_params('scs', 'cert', True, False, CERT_NUM_CONSTRAINTS, SEGMENT_SIZE)
+        cswVk  = cswMcTest.generate_params('scs', 'csw', True, False, CSW_NUM_CONSTRAINTS, SEGMENT_SIZE)
```

Comment:
What about using parameters name to improve readability?

```suggestion
        certVk = certMcTest.generate_params('scs', 'cert', constant = True, keyrot = False, CERT_NUM_CONSTRAINTS, SEGMENT_SIZE)
        cswVk  = cswMcTest.generate_params('scs', 'csw', constant = True, keyrot = False, CSW_NUM_CONSTRAINTS, SEGMENT_SIZE)
```

Reply:
Good point, done with f663cac6ada8f2b0e099497b23ac96371e611b14